examples/text_labels_and_annotations/mathtext_examples.py

```diff
@@ -92,12 +91,11 @@ def doall():
                          color=fill_color, alpha=0.5)
         plt.annotate(title,
                      xy=(0.07, baseline - 0.3 * line_axesfrac),
-                     xycoords='data', color=mpl_grey_rvb, weight='bold')
```

Comment:
I don't think its helpful to drop all the `xycoord='data'` calls.  Its good for users to understand that there are different co-oridnates that annotate can be expressed in...

Reply:
But the point is that "data" *is* the default for annotate() just like it is the default for nearly every other function (e.g. in particular text()).
I purposefully left the xycoords=... in the main annotation demo and tutorials (where the coordinates systems are explained), but outside of them I think these are just noise.
(And again I'd rather have people learn to use ax.transData/ax.transAxes which *also* works for text(), instead of an annotate()-specific API.)